### ISSUE TITLE
Editorial alt text is ambiguous and used on decorative images

### ISSUE THREAD

**Issue by LaurenFBaily** [wellcomecollection/wellcomecollection.org#12664]
### What

From 2025 accessibility audit: _Alt text which is ambiguous and sometimes confusing is used for decorative images. In most cases the caption gives necessary information so the image should be marked as decorative._

For a screenreader user looking for information on getting to the venue, the existence of alt text on that image (as well as the detail/length) is a barrier to accessing the page content.

### Definition of done

- [x] Editorial team have been informed and further guidance provided if necessary
- [ ] Timeline agreed to resolve issue across site

### Where

Example is alt text on the lead image on this page: https://wellcomecollection.org/visit-us/getting-here
